conslogging/conslogging.go

```diff
@@ -266,6 +272,67 @@ func (cl ConsoleLogger) PrefixColor() *color.Color {
 	return cl.color(c)
 }
 
+// Prints a GitHub Actions summary message to GITHUB_STEP_SUMMARY
+func (cl *ConsoleLogger) PrintGHASummary(message string) {
+	if !cl.isGitHubActions {
+		return
+	}
+
+	path := os.Getenv("GITHUB_STEP_SUMMARY")
```

Comment:
where is this being set/by what?

Reply:
This part is a bit magic. When you run in GHA it sets that to a file that it uses to populate the summary page. So in GHA runner, GITHUB_STEP_SUMMARY is a file path to some temp uuid file and also GITHUB_ACTIONS is always equal to "true".

Reply:
oh gotcha, thanks! please disregard my comments about renaming the `GITHUB_ACTIONS` env var.

Question, do you know if this is set by the GH runner or something that is set to the runner?
Just wondering if it's something we need to set in our own GH runner and for Nacho to be aware of.

Reply:
@idelvall are these set in your GHA runner by chance?

Reply:
I will check. I am pretty sure the latter it's set

Reply:
One thing to have in mind is that the GHA might mix earthly commands and others. Wondering if these changes would give too much relevance to the Earthly code